<?xml version="1.0"?>
<launch>
  <group ns="mcr_perception">
    <node pkg="mcr_perception_selectors" type="object_selector" name="object_selector" output="screen" respawn="false">
      <remap from="~input/object_list" to="/mcr_perception/object_detector/object_list" />
      <param name="frame_rate" type="int" value="5" />
    </node>
  </group>
</launch>
